internal/exec/stages/mount/mount.go

```diff
@@ -68,7 +68,14 @@ func (stage) Name() string {
 	return name
 }
 
+func isNoOp(config types.Config) bool {
+	return len(config.Storage.Filesystems) == 0
+}
+
 func (s stage) Run(config types.Config) error {
+	if isNoOp(config) {
```

Comment:
The new check here and in umount stage is redundant, so normally I'd be inclined to omit it.  I assume this is needed for future work, though.

Reply:
Yeah, if `isNoOp` isn't going to become part of the stage API, I feel like they're slightly misleading and would prefer to drop.

Reply:
Updated!